opentracing-api/src/main/java/io/opentracing/tag/Tags.java

```diff
@@ -18,7 +18,8 @@
  * semantic information about the spans. Tracers may expose additional features based on these
  * standardized data points. Tag names follow a general structure of namespacing.
  *
- * @see http://opentracing.io/data-semantics/
+ * @see <a href="http://opentracing.io/documentation/pages/spec.html">http://opentracing.io/documentation/pages/spec
```

Comment:
This was wrong. I'm not sure which link should be used. This one or:

https://github.com/opentracing/specification/blob/master/data_conventions.yaml